### ISSUE TITLE
[Felt privacy-Unified panel] [Accessibility] - Voice over doesn't read the site name

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#23322]
**Steps to reproduce:**

1. Turn voice over ON.
1. Access a website and tap on the lock icon to open the panel.
1. Swipe to one of the submenus. 
1. Listen.

**Actual results**

VoiceOver does not read the site name when swiping in submenus.

**Expected results**

The site name is correctly read.

**Device & build information**

* Device: iPhone 14 Pro Max (iOS 16.2)
* Build version: v9000(47935)
* First seen version: ?

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-10661)

**Comment by data-sync-user** [comment]
➤ Diana Andreea Barladeanu commented:

Verified as fixed on v136(50708), with iPhone 15 (17.6).
